build.sbt

```diff
@@ -1403,6 +1405,7 @@ def scriptedSettings: Seq[Setting[_]] =
   Seq(scriptedLaunchOpts += s"-Dproject.version=${version.value}") ++
     Seq(
       scripted := scripted.tag(Tags.Test).evaluated,
+      scriptedBufferLog := false,
```

Comment:
So we can have more information about all scripted tests runs. A missing thing here is that it would be good to have `logback.xml` configuration for each scripted test.

Reply:
raised https://github.com/lagom/lagom/issues/2187